libs/testing/src/user.session.ts

```diff
@@ -269,4 +271,23 @@ export class UserSession {
       timedOut = timeout < +new Date() - startTime;
     } while (runningJobs > 0 && !timedOut);
   }
+
+  public async applyChanges(where: any = {}) {
```

Comment:
Love that you extracted this one here, but would be great if don't use any here. We can do: Partial<ChangeEntity> or something similar.